lib/broadside/deploy.rb

```diff
@@ -9,12 +9,14 @@ class Deploy
       :command,
       :instance,
       :lines,
+      :rollback,
+      :scale,
       :tag,
       :target
     )
 
-    def initialize(target, options = {})
-      @target   = target
+    def initialize(target_name, options = {})
+      @target   = Broadside.config.target_from_name!(target_name)
```

Comment:
I like having this validation call here rather than repeated in a bunch of other places